src/lib/client.ts

```diff
@@ -26,24 +26,27 @@ const SUNIT = 1000000000000000000n
 const NETWORK_RPC = process.env.PUBLIC_API_WS || "ws://localhost:9944"
 const LOCAL_RPC = process.env.LOCAL_API_WS || "ws://localhost:9944"
 const appsLink = "https://polkadot.js.org/apps/?rpc=" + NETWORK_RPC + "#/explorer/query/"
+
+const publicApi = new ApiPromise({
```

Comment:
Why not doing this in constructor, so you have control over when it happens instead of running on file inclusion?

Reply:
I am afraid that is the actual fix for the problem. Originally API was assigned directly as a class property and it was causing the problem. Apparently, Quasar or Vue use `Proxy` somewhere under the hood - https://github.com/polkadot-js/api/issues/4656 Using constructor does not solve this.

Reply:
You need to use async contructor and wait for API to actually initialize before you use it, did you try that to make sure it is actually working?

If contructors are broken with Quasar or Vue then bad things will be literally everywhere, I don't believe that is the case, but async initialization of Polkadot.js's API might easily be.

Reply:
Async constructor may work, but it will require some refactoring (not just Client class), this was intended as a quick fix

Reply:
Can you at least not refactor things to use global variable and assign property of the class to the constant? And TODO there to refactor to async constructor.

Reply:
I made an attempt to fix this with as minimal effort as possible, but it requires much more. Please, check